core/store/src/main/java/mil/nga/giat/geowave/core/store/operations/config/AddIndexGroupCommand.java

```diff
@@ -17,34 +20,100 @@
 import mil.nga.giat.geowave.core.cli.operations.config.options.ConfigOptions;
 import mil.nga.giat.geowave.core.store.operations.remote.options.IndexGroupPluginOptions;
 import mil.nga.giat.geowave.core.store.operations.remote.options.IndexPluginOptions;
+import mil.nga.giat.geowave.core.cli.parser.ManualOperationParams;
 
 @GeowaveOperation(name = "addindexgrp", parentOperation = ConfigSection.class)
 @Parameters(commandDescription = "Create an index group for usage in GeoWave")
 public class AddIndexGroupCommand extends
 		DefaultOperation implements
 		Command
 {
+	private static int SUCCESS = 0;
```

Comment:
Try to use exceptions when possible, and catch them in computeResults (so they throw the correct error code). Set SetCommand for an example.